doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
+or local communications (domain sockets on UNIX, named pipes on Windows).
```

Comment:
:S just got enlightened by @addaleax , the stuff in this module is stream-based, UDP(dgram) is not.